.circleci/config.yml

```diff
@@ -54,17 +54,12 @@ jobs:
             - .emscripten_ports/
             - .emscripten
 
-  test-other-am:
+  test-other:
     <<: *test-defaults
     environment:
-      - TEST_TARGET=other.test_a* other.test_b* other.test_c* other.test_d* other.test_e* other.test_f* other.test_g* other.test_h* other.test_i* other.test_j* other.test_k* other.test_l* other.test_m* skip:other.test_bad_triple skip:other.test_emcc_v
-      # TODO: remove skips after fastcomp update on travis for 1.37.23
-
-  test-other-nz:
-    <<: *test-defaults
-    environment:
-      - TEST_TARGET=other.test_n* other.test_o* other.test_p* other.test_q* other.test_r* other.test_s* other.test_t* other.test_u* other.test_v* other.test_w* other.test_x* other.test_y* other.test_z* skip:other.test_native_link_error_message
-      # TODO: remove skips after fastcomp update on travis for 1.37.23
+      - TEST_TARGET=other skip:other.test_bad_triple skip:other.test_native_link_error_message skip:other.test_emcc_v skip:other.test_failing_alloc
```

Comment:
why is `emcc_v` skipped?

Reply:
It has been skipped since its birth because it tends to fail whenever an SDK version update happens: https://github.com/kripken/emscripten/pull/5883